More grammar fixes - Heliarch and Lunarium intros (#9633)
endless-sky/endless-sky: data/coalition/lunarium intro.txt

	destination "Factory of Eblumab"
	on offer
		conversation
			`An Arach stops you by the spaceport and invites you to a free meal at a local bar. "A business proposal for you, my employers have."`
			choice
				`	"Very well, lead the way."`
					goto bar
				`	"Can't you just give me the details here? I'm not hungry."`

> 			`	He seems somewhat disappointed, but agrees. "A messenger from the great House Bebliss, I am. Expressed a wish to see you, my employers have, for a great aid to one of their projects, you would be."`

Reviewer: ```suggestion
			`	He seems somewhat disappointed but agrees. "A messenger from the great House Bebliss, I am. Expressed a wish to see you, my employers have, for a great aid to one of their projects, you would be."`
```

		conversation
			`You follow the coordinates given to you and land on top of a mountain at a set of private hangars. Nearby is an estate, covered by a dome similar to the one over the spaceport city. You are received by who you assume are Arachi butlers and maids, given their tidy, well-kept uniforms, and follow one of them into the mansion.`
			`	They bring you to a large, wide room where three preoccupied Arachi are pouring over schematics and blueprints scattered across a table.`

> 			`	"Captain <first> <last>!" One of them comes to greet you, after the servant announces your arrival. "A rare opportunity, it is, such an exquisite guest, to entertain. Care for a drink, would you?" Without waiting for a response, one of the maids brings over a platter with several cups, and you're pleasantly surprised to discover familiar smells like coffee and tea coming from the silver dish. You notice that there are also some alcoholic beverages as well as just plain water in the middle of the platter.`

Reviewer: ```suggestion
			`	"Captain <first> <last>!" One of them comes to greet you after the servant announces your arrival. "A rare opportunity, it is, such an exquisite guest, to entertain. Care for a drink, would you?" Without waiting for a response, one of the maids brings over a platter with several cups, and you're pleasantly surprised to discover familiar smells like coffee and tea coming from the silver dish. You notice that there are also some alcoholic beverages as well as just plain water in the middle of the platter.`
```

				`	"Human space was invaded by a species known as the Pug."`
			label pug
			`	Thankfully, the ones that were downing drinks weren't that close when you said the word "Pug," or else you'd need a new set of clothes. One of them signals for a maid to clean up their simultaneous spit takes, and the Arach by the blueprints finally comes closer.`

> 			`	"'Pug,' were they called?" she asks as the maid behind her quickly scrubs the floor. You affirm her question, and she calls for a servant, asking them to ready a ship. She whispers something to the other two members of House Bebliss, which the translation devices don't pick up, and starts gathering her belongings. "To forgive me, I ask you, Captain, but leave at once, I must."`

Reviewer: ```suggestion
			`	"'Pug,' were they called?" she asks as the maid behind her quickly scrubs the floor. You answer in the affirmative, and she calls for a servant, asking them to ready a ship. She whispers something to the other two members of House Bebliss, which the translation devices don't pick up, and starts gathering her belongings. "To forgive me, I ask you, Captain, but leave at once, I must."`
```

Reviewer: I agree, it does sound kind of verbose. On the other hand, "affirming" someone question doesn't feel like correct English. What do you think?

	blocked `You have reached <origin>, but you need <capacity> in order to take on the next mission. Return here when you have the required space free.`
	on offer
		conversation

> 			`You look around for Freya, asking the local dock workers where the people researching the Pug tech left on the planet are. They direct you to a quieter, less crowded place, where some warehouse complexes have been set up. Some are painted in the usual Syndicate colors, some are guarded by a few Navy troops, and a Free Worlds flag flies by the entrance of the one you enter. Inside, you see groups of people surrounding the various artifacts the Pug left behind. Some of them are running complicated looking tests, but it seems that they are mostly discussing what to do with the artifacts next. One of the graviton transmitters sits opposite to the entrance, half intact, half dismantled, with an array of scientists poking and prodding the machinery inside.`

Reviewer: ```suggestion
			`You look around for Freya, asking the local dock workers to point you in the direction of the people researching the Pug tech whom still remain on the planet. They direct you to a quieter, less crowded place where some warehouse complexes have been set up. Some are painted in the usual Syndicate colors, some are guarded by a few Navy troops, and a Free Worlds flag flies by the entrance of the one you enter. Inside, you see groups of people surrounding the various artifacts that the Pug left behind. Some of them are running complicated looking tests, but it seems that they are mostly discussing what to do with the artifacts next. One of the graviton transmitters sits opposite to the entrance, half intact, half dismantled, with an array of scientists poking and prodding the machinery inside.`
```

Reviewer: That works too. I only included that last bit to retain as much of the original meaning on my suggestion as possible, but I agree that it's better without it.

Reviewer: ```suggestion
			`You look around for Freya, asking the local dock workers to point you in the direction of the people researching the Pug tech. They direct you to a quieter, less crowded place, where some warehouse complexes have been set up. Some are painted in the usual Syndicate colors, some are guarded by a few Navy troops, and a Free Worlds flag flies by the entrance of the one you enter. Inside, you see groups of people surrounding the various artifacts that the Pug left behind. Some of them are running complicated looking tests, but it seems that they are mostly discussing what to do with the artifacts next. One of the graviton transmitters sits opposite to the entrance, half intact, half dismantled, with an array of scientists poking and prodding the machinery inside.`
```
